srcpkgs/libnotify/template

```diff
@@ -9,11 +9,11 @@ configure_args="-Dintrospection=$(vopt_if gir enabled disabled)
 hostmakedepends="pkg-config glib-devel libxslt docbook-xsl-ns"
 makedepends="libglib-devel libpng-devel gdk-pixbuf-devel gtk+3-devel"
 short_desc="Desktop notification library"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
 license="LGPL-2.1-or-later"
 homepage="http://library.gnome.org/devel/notification-spec/"
```

Comment:
404

Reply:
[this](https://developer-old.gnome.org/notification-spec/) looks close to what that page was, but [this](https://gitlab.gnome.org/GNOME/libnotify) is probably a better homepage

Reply:
Went ahead and changed all pages to gitlab. added changelog aswell